salesforce/errors.go

```diff
@@ -70,3 +71,24 @@ func (c *Connector) interpretJSONError(res *http.Response, body []byte) error {
 	// No known errors, just do the normal error handling logic
 	return common.InterpretError(res, body)
 }
+
+func (c *Connector) HandleError(err error) error {
+	switch {
+	case errors.Is(err, common.ErrAccessToken):
+		// Retryable, so just log and retry
+		slog.Warn("Access token invalid, retrying", "error", err)
```

Comment:
Can we remove this warning line? since we're not actually retrying yet (it's still a TODO)